go/vt/mysqlctl/replication.go

```diff
@@ -612,9 +612,48 @@ func (mysqld *Mysqld) GetPreviousGTIDs(ctx context.Context, binlog string) (prev
 	return previousGtids, nil
 }
 
+var ErrNoSemiSync = errors.New("semisync plugin not loaded")
+
+func (mysqld *Mysqld) SemisyncType(ctx context.Context) mysql.SemiSyncType {
```

Comment:
Sorry, can you capitalize `sync` consistently across the funcs in this file? Other instances are at lines 624 and 634

Reply:
Updated!

Reply:
You missed the one on line 634, and in fact now I see this also on line 644.

Reply:
Doh, fixed!